### ISSUE TITLE
Link external URLs

### ISSUE THREAD

**Issue by Kikobeats** [docpress/docpress#138]
Sometimes a need to reference a external URL from the sidebar.

For do that, basically I add this markup:

```
* [JS MythBusters](/README.md)
       	* [Send your tip!](/send-tip.md)
```

when `send-tip.md` is just a empty file necessary for be compiled by docpress, and later using JS I change the `href` of the URL generated.

Could be nice support URL's directly from Docpress.

**Comment by rstacruz** [comment]
Yes, this seems like a great addition. It'd be nice to directly link to GitHub, StackOverflow, Hexdocs.pm, and other externals.

**Comment by kzap** [comment]
does it already work directly on external markdown files? sometimes i need to reference MD files from other repositories, but then if they are private that would be an issue...
